app/Http/Controllers/Ajax/AvailabilityMapController.php

```diff
@@ -33,7 +33,7 @@ class AvailabilityMapController extends Controller
 {
     public function setView(Request $request)
     {
-        $this->validate($request, [
+        $request->validate([
```

Comment:
This no longer throws exceptions if I remember...  Wouldn't it be better to add `use ValidatesRequests;`

Reply:
Ok, I checked and it does throw exceptions, all good :)